src/pages/Animal/MyPets/index.tsx

```diff
@@ -22,10 +24,12 @@ const MyPets = (): JSX.Element => {
   }, [navigation]);
 
   const fetchPets = (): void => {
-    const { currentUser } = auth();
-    const user = firestore().collection('users').doc(currentUser?.uid);
+    const currentUser = userAPI.currentUser();
+    const user = userAPI.userFirestoreDocument(currentUser?.uid);
```

Comment:
Here the same: `Users.get(currentUser?.uid)`

Reply:
Fixed by new commit (although, in this case, we need the document from the user, not the `.get()` method).